docs/send-data/opentelemetry-collector/data-source-configurations/collect-logs.md

```diff
@@ -212,6 +216,67 @@ Configuration details:
 * **service**:
   * `logs/custom_files:` Pipeline glues together the receivers with the processors and the exporters.
 
+
+### Collect from Custom channels (Powershell, Sysmon)
+
+
+Following configuration demonstrates:
+
+1. **Collect**: Collect Sysmon logs.
+2. **Transform**: Set `_sourceCategory` field to `windows_event_log_prod_sysmon`.
+3. **Export**: Send data to authenticated Sumo Logic organization.
+
+```yaml
+receivers:
+  windowseventlog/sysmon/localhost/1690233479:
+    channel: Microsoft-Windows-Sysmon/Operational 
+
+processors:
+  resource/windows_resource_attributes/localhost/1690233479:
+    attributes:
+      - key: _sourceCategory
+        value: windows_event_log_prod_sysmon
+        action: insert
+
+service:
+  pipelines:
+    logs/windows/localhost/1690233479:
+      receivers:
+        -  windowseventlog/sysmon/localhost/1690233479
+      processors:
+        - memory_limiter
+        - resource/windows_resource_attributes/localhost/1690233479
+        - batch
+      exporters:
+        - sumologic
+```
+
+1. Create a file in folder `C:\ProgramData\Sumo Logic\OpenTelemetry Collector\config\` with name `sysmon_windows.yaml`.
+2. Paste the above content into the file.
+3. Restart collector with following command:
+   ```bash title="Windows"
+   Restart-Service -Name OtelcolSumo
+   ```
```

Comment:
@ajaiswals , should these steps be numbered 4, 5, 6?

Reply:
@kimsauce  - we can keep them 1,2,3. Thats how we have kept for other sections as well.

1,2,3 before this section is to give an overview what we will do and steps after config.yaml is the actual steps.

Probable we can add a heading after snippet. "Steps to Apply configuration" ?